addons/binding/org.openhab.binding.airquality/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,177 @@
+				<label>API Key</label>
+				<description>Data-platform token to access the AQIcn.org service</description>
+			</parameter>
+			<parameter name="location" type="text" required="false"
+				pattern="^[-+]?([1-8]?\d(\.\d+)?|90(\.0+)?)[,|;]\s*[-+]?(180(\.0+)?|((1[0-7]\d)|([1-9]?\d))(\.\d+)?)$">
+				<label>Location</label>
+				<description>Your geo coordinates separated with comma or semicolon (e.g. "37.8;-122.4").</description>
+			</parameter>
+			<parameter name="stationId" type="integer" required="false">
+				<label>Station ID</label>
+				<description>Fill only in case you want to receive data from the specific station</description>
+			</parameter>
+			<parameter name="refresh" type="integer" min="30" required="false">
+				<label>Refresh interval</label>
+				<description>Specifies the refresh interval in minutes.</description>
+				<default>60</default>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+	<channel-type id="aqiLevel">
+		<item-type>Number</item-type>
+		<label>Air Quality Index</label>
+		<description></description>
+		<category>Air Quality Index</category>
+		<state readOnly="true" pattern="%d"></state>
+	</channel-type>
+
+	<channel-type id="aqiDescription">
+		<item-type>String</item-type>
+		<label>AQI Description</label>
+		<description></description>
+		<category>AQI Description</category>
+		<state readOnly="true" pattern="%s"></state>
+	</channel-type>
+
+	<channel-type id="pm25">
+		<item-type>Number</item-type>
+		<label>PM2.5</label>
+		<description>Fine particles pollution level</description>
+		<category>PM2.5</category>
+		<state readOnly="true" pattern="%d μg/m&#179;"></state>
+	</channel-type>
+
+	<channel-type id="pm10">
+		<item-type>Number</item-type>
+		<label>PM10</label>
+		<description>Coarse dust particles pollution level</description>
+		<category>PM10</category>
+		<state readOnly="true" pattern="%d μg/m&#179;"></state>
+	</channel-type>
+
+	<channel-type id="o3">
+		<item-type>Number</item-type>
+		<label>O3</label>
+		<description>Ozone level</description>
+		<category>O3</category>
+		<state readOnly="true" pattern="%.1f μg/m&#179;"></state>
+	</channel-type>
+
+	<channel-type id="no2">
+		<item-type>Number</item-type>
+		<label>NO2</label>
+		<description>Nitrogen dioxide level</description>
+		<category>NO2</category>
+		<state readOnly="true" pattern="%.1f μg/m&#179;"></state>
+	</channel-type>
+
+	<channel-type id="co">
+		<item-type>Number</item-type>
+		<label>CO</label>
+		<description>Carbon monoxide level</description>
+		<category>CO</category>
+		<state readOnly="true" pattern="%.1f μg/m&#179;"></state>
+	</channel-type>
+
+	<channel-type id="locationName" advanced="true">
+		<item-type>String</item-type>
+		<label>Location</label>
+		<description>Nearest measuring station location</description>
+		<category>Location</category>
+		<state readOnly="true" pattern="%s"></state>
+	</channel-type>
+
+	<channel-type id="locationUrl" advanced="true">
```

Comment:
Is this channel really useful? We do not support links in item states, so I do not know, how this should be used in any meaningful way?